### PR TITLE
WP-192: Adding getBalances call

### PR DESCRIPTION
**Pull request by VKCR**

### CODE REVIEW

File: test/sample/MemberGetBalancesSample.spec.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+import 'babel-regenerator-runtime';
+import CreateMemberSample from '../../src/sample/CreateMemberSample';
+import LinkMemberAndBankSample from '../../src/sample/LinkMemberAndBankSample';
+import MemberGetBalancesSample from '../../src/sample/MemberGetBalancesSample';
+
+describe('MemberGetBalancesSample test', () => {
+    it('Should run the sample', async () => {
+        const member = await CreateMemberSample();
+	await LinkMemberAndBankSample(member);
```
Comment (mariano54) [inline test/sample/MemberGetBalancesSample.spec.js]: indent

File: src/http/AuthHttpClient.js [hunk 2]
```diff
@@ -504,6 +504,28 @@ class AuthHttpClient {
...
+        this.useKeyLevel(keyLevel);
+        var url = '/accounts/balance?';
+
+        for (var i = 0; i < accountIds.length; ++i) {
+            url += 'account_id=' + accountIds[i] + '&';
+        }
+
+        const request = {
+            method: 'get',
+            url: url,
```
Comment (mariano54) [inline src/http/AuthHttpClient.js]: you can just write: url,

File: src/http/AuthHttpClient.js [hunk 3]
```diff
@@ -504,6 +504,28 @@ class AuthHttpClient {
...
+     *
+     * @param {Array} accountIds - array of accountIds
+     * @param {string} keyLevel - key level
+     * @return {Object} response - response to the API call
+     */
+    async getBalances(accountIds, keyLevel) {
+        this.useKeyLevel(keyLevel);
+        var url = '/accounts/balance?';
+
+        for (var i = 0; i < accountIds.length; ++i) {
```
Comment (lahosken) [inline src/http/AuthHttpClient.js]: Might be able to impress your functional programmer friends via accountIds.map(...)

File: src/http/AuthHttpClient.js [hunk 4]
```diff
@@ -504,6 +504,28 @@ class AuthHttpClient {
...
+            url += 'account_id=' + accountIds[i] + '&';
```
Comment (lahosken) [inline src/http/AuthHttpClient.js]: JS arrays have a join nowadays: [ 'x=foo', 'x=bar' ].join('&') // gives "x=foo&x=bar"

File: src/main/Member.js [hunk 5]
```diff
@@ -770,7 +770,7 @@ export default class Member {
      *
      * @param {string} accountId - id of the account
      * @param {string} keyLevel - key level
-     * @return {Promise} balance - Promise of balance object
+     * @return {Promise} balance - Promise of get balance response object
```
Comment (lahosken) [inline src/main/Member.js]: good catch

File: test/sample/MemberGetBalancesSample.spec.js (hunk 1 above)
Comment (VKCR) [reply test/sample/MemberGetBalancesSample.spec.js]: done

File: src/http/AuthHttpClient.js (hunk 2 above)
Comment (VKCR) [reply src/http/AuthHttpClient.js]: I feel like it would just make it more confusing.

File: src/sample/MemberGetBalancesSample.js [hunk 6]
```diff
@@ -0,0 +1,24 @@
+import config from "../config.json";
+
+/**
+ * Gets a logged-in member's account balances two ways.
```
Comment (lahosken) [inline src/sample/MemberGetBalancesSample.js]: s/two ways/another way/

File: test/sample/MemberGetBalanceSample.spec.js [hunk 7]
```diff
@@ -10,7 +10,7 @@ import MemberGetBalanceSample from '../../src/sample/MemberGetBalanceSample';
 describe('MemberGetBalanceSample test', () => {
     it('Should run the sample', async () => {
         const member = await CreateMemberSample();
-	await LinkMemberAndBankSample(member);
+	    await LinkMemberAndBankSample(member);
```
Comment (lahosken) [inline test/sample/MemberGetBalanceSample.spec.js]: (in this github web-diff view, both the old and the new indentation look off.)

File: test/sample/MemberGetBalancesSample.spec.js [hunk 8]
```diff
@@ -0,0 +1,18 @@
...
+
+import 'babel-regenerator-runtime';
+import CreateMemberSample from '../../src/sample/CreateMemberSample';
+import LinkMemberAndBankSample from '../../src/sample/LinkMemberAndBankSample';
+import MemberGetBalancesSample from '../../src/sample/MemberGetBalancesSample';
+
+describe('MemberGetBalancesSample test', () => {
+    it('Should run the sample', async () => {
+        const member = await CreateMemberSample();
+	    await LinkMemberAndBankSample(member);
```
Comment (lahosken) [inline test/sample/MemberGetBalancesSample.spec.js]: (indentation looks funny. maybe run an extra `yarn link` for luck

File: src/http/AuthHttpClient.js (hunk 3 above)
Comment (VKCR) [reply src/http/AuthHttpClient.js]: done

File: src/http/AuthHttpClient.js (hunk 4 above)
Comment (VKCR) [reply src/http/AuthHttpClient.js]: done

File: src/sample/MemberGetBalancesSample.js (hunk 6 above)
Comment (VKCR) [reply src/sample/MemberGetBalancesSample.js]: done

File: test/sample/MemberGetBalanceSample.spec.js (hunk 7 above)
Comment (VKCR) [reply test/sample/MemberGetBalanceSample.spec.js]: Yeah I think I messed up the indent the first time I tried to fix it, but it looks ok now.

**Pull request merged**